src/main/java/frc/robot/commands/AutoScoreAndBalance.java

```diff
@@ -18,12 +25,19 @@ public class AutoScoreAndBalance extends SequentialCommandGroup {
               Cal.SwerveSubsystem.MAX_LINEAR_SPEED_METERS_PER_SEC,
               Cal.SwerveSubsystem.MAX_LINEAR_ACCELERATION_METERS_PER_SEC_SQ));
 
-  public AutoScoreAndBalance(boolean isFirstPath, Lift lift, DriveSubsystem drive) {
-    // TODO update this since scoreGamePiece and doneScoring are stubbed
+  public AutoScoreAndBalance(boolean isFirstPath, Lift lift, DriveSubsystem drive, ScoringLocationUtil scoringLocationUtil) {
     addCommands(
-        new RunCommand(lift::scoreGamePiece, lift).until(lift::doneScoring),
-        drive.followTrajectoryCommand(traj, isFirstPath),
-        new AutoChargeStationBalance(drive));
+      new InstantCommand(()->scoringLocationUtil.toggleMiddleGrid()), //is this needed??
+      new InstantCommand(() -> scoringLocationUtil.setScoreCol(ScoreCol.RIGHT)),
+      new InstantCommand(() -> scoringLocationUtil.setScoreHeight(ScoreHeight.HIGH)),
+      new InstantCommand(lift::ManualPrepScoreSequence, lift),
+      new WaitUntilCommand(() -> lift.atPosition(LiftPosition.PRE_SCORE_HIGH_CONE)),
+      new InstantCommand(lift::startScore, lift),
+      new WaitUntilCommand(() -> lift.atPosition(LiftPosition.SCORE_HIGH_CONE)),
+      new finishScore(lift),
+      drive.followTrajectoryCommand(traj, isFirstPath),
+      new AutoChargeStationBalance(drive),
+      new InstantCommand(()->scoringLocationUtil.toggleMiddleGrid())); //i assume i should toggle this back as we want it to by default be at false
```

Comment:
`AutoChargeStationBalance` does not end at the moment (it will be interrupted once auton ends), so `toggleMiddleGrid` is never getting called. Swapping the two should fix it.